pymess/models/common.py

```diff
@@ -29,6 +34,48 @@ def create_from_related_objects(self, *related_objects):
             for related_object in related_objects
         ]
 
+    def _get_related_object_pks(self, *related_objects):
+        return [related_object.pk for related_object in related_objects]
+
+    def _get_related_objects_qs_kwargs(self, *related_objects):
+        return reduce(OR, (
+            Q(**{
+                'object_id': obj.pk,
+                'content_type': ContentType.objects.get_for_model(obj)
+            }) for obj in related_objects
+        ))
+
+    def update_related_objects(self, *related_objects, limit_to_model=None):
```

Comment:
when limit_to_model is None and you send

update_related_objects(related_object)

all other relations will be removed? It seems to be insecure

Reply:
In that case you assume you are sending all related objects there should be (with mixed content_type), not sure if it makes sense.

Reply:
ok let it be